barchart/app.js

```diff
@@ -68,57 +70,53 @@ angular.module('myApp', ['ngStorage', 'ngMaterial', 'ngMessages' ,'ngSanitize',
             $interval.cancel(promise);
             vm.wallTweets = [];
             vm.limit = 25
-            
+
             // clear form
             // $scope.query={};
-            
+
             // first call
             queryService.queryTweets(type, term, dateFrom, dateTo, maxHashtags, count);
 
             // poll every few minutes
-            promise = $interval(function(){
-                queryService.queryTweets(type, term, dateFrom, dateTo, maxHashtags, count);
+            promise = $interval(function () {                queryService.queryTweets(type, term, dateFrom, dateTo, maxHashtags, count);
```

Comment:
Line is longer than allowed. (136 > 129)

*LineLengthBear, severity NORMAL, section `default`.*